grammar.js

```diff
@@ -1241,11 +1290,79 @@ module.exports = grammar({
       ),
     ),
 
+    alter_type: $ => seq(
+      $.keyword_alter,
+      $.keyword_type,
+      $.identifier,
+      choice(
+        $.change_ownership,
+        $.set_schema,
+        $.rename_object,
+        seq(
+          $.keyword_rename,
+          $.keyword_attribute,
+          $.identifier,
+          $.keyword_to,
+          $.identifier,
+          optional($._cascade_restrict)
+        ),
+        seq(
+          $.keyword_add,
+          $.keyword_value,
+          optional($._if_not_exists),
+          $.identifier,
```

Comment:
this should be `literal`, identifiers cannot be specified as enum values (even trying to get cute with it in a procedure)

```sql
[local] dian#dian= create procedure test(newval text) language plpgsql as $$ begin
[local] dian#dian$ alter type boxes add value if not exists newval;
[local] dian#dian$ return;
[local] dian#dian$ end;
[local] dian#dian$ $$;
ERROR:  syntax error at or near "newval"
LINE 2: alter type boxes add value if not exists newval;
                                                 ^
Time: 23.887 ms
```